### PR TITLE
blog: add v0.17.2.3 release notes

### PR DESCRIPTION
**Pull request by selsta**

### CODE REVIEW

File: _posts/2021-08-31-monero-0.17.2.3-released.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+- Update randomx to v1.1.9 (#7734)
+- Fix boost 1.76.0 compatibility(#7736)
+- Fix rescan\_bc keep\_ki option (#7680)
+- Minor bug fixes
+
+The complete list of changes is [available on GitHub](https://github.com/monero-project/monero/compare/v0.17.2.0...v0.17.2.3), along with [the source code](https://github.com/monero-project/monero/tree/v0.17.2.3).
+
+# Contributors for this Release
+
+This release was the direct result of 16 who worked, largely unpaid and altruistically, to put out 87 commits containing 838 new lines of code. We'd like to thank them very much for their time and effort. In no particular order they are:
```
Comment (erciccione) [inline _posts/2021-08-31-monero-0.17.2.3-released.md:39]:
```suggestion
This release was the direct result of 16 people who worked, largely unpaid and altruistically, to put out 87 commits containing 838 new lines of code. We'd like to thank them very much for their time and effort. In no particular order they are:
```

File: _posts/2021-08-31-monero-0.17.2.3-released.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+   <img src="{{ page.image }}" width="350px">
+ </div>
+
+# Overview
+
+This is the v0.17.2.3 point release of the Monero software. This release contains a decoy selection bug fix that improves privacy.
+
+Some highlights of this point release are:
+
+- Apply gamma distribution from chain tip when selecting decoys (#7822)
```
Comment (erciccione) [inline _posts/2021-08-31-monero-0.17.2.3-released.md:20]: If we give the number of PR, would be good to provide a link to them.

File: _posts/2021-08-31-monero-GUI-0.17.2.3-released.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+# Overview
+
+This is the v0.17.2.3 minor point release of the Monero GUI software. This release contains a decoy selection bug fix that improves privacy.
+
+[The latest CLI release notes can be found on the precedent blog post]({{ site.baseurl_root }}/2021/04/09/monero-0.17.2.0-released.html)
+
+Some highlights of this minor release are:
+
+- Add payment request functionality to receive page (#3650)
```
Comment (erciccione) [inline _posts/2021-08-31-monero-GUI-0.17.2.3-released.md:22]: See my comment above abotu adding links to PRs.

**Comment by selsta** [comment]
done

**Review by rating89us — APPROVED** [review]

**Review by SamsungGalaxyPlayer — APPROVED** [review]

**Comment by selsta** [comment]
Sorry, had to force push once more. Added a missing space.

File: _posts/2021-08-31-monero-GUI-0.17.2.3-released.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+<div align="center">
+    <img src="{{ page.image }}" width="350px">
+</div>
+
+# Overview
+
+This is the v0.17.2.3 minor point release of the Monero GUI software. This release contains a decoy selection bug fix that improves privacy.
+
+[The latest CLI release notes can be found on the precedent blog post]({{ site.baseurl_root }}/2021/04/09/monero-0.17.2.0-released.html)
```
Comment (rating89us) [inline _posts/2021-08-31-monero-GUI-0.17.2.3-released.md:18]:
```suggestion
[The latest CLI release notes can be found on the precedent blog post]({{ site.baseurl_root }}/2021/04/09/monero-0.17.2.0-released.html).
```

**Review by erciccione — APPROVED** [review]

File: _posts/2021-08-31-monero-GUI-0.17.2.3-released.md (hunk 4 above)
Comment (selsta) [reply _posts/2021-08-31-monero-GUI-0.17.2.3-released.md]: Fixed the link from v0.17.2.0 to v0.17.2.3 and added missing dot.

**Review by erciccione — APPROVED** [review]

**Comment by binaryFate** [comment]
Hashes match mine

**Pull request merged**